db/seeds.d/050-taxonomies.rb

```diff
@@ -8,7 +8,11 @@
     taxonomy.without_auditing do
       tax_name = ENV.fetch("SEED_#{taxonomy.to_s.upcase}", "Default #{taxonomy}")
       tax = taxonomy.create!(name: tax_name)
-      associations = taxonomy.reflect_on_all_associations.reject {|assoc| skip_associations.include?(assoc.name)}
+      associations = taxonomy.reflect_on_all_associations.select do |assoc|
```

Comment:
How about applying a De Morgan here and making it a reject instead? Not really crucial, but might make it a bit clearer to understand.

Reply:
works for me!